jfrog_platform/README.md

```diff
@@ -0,0 +1,64 @@
+# Artifactory and Xray Logging Analytics with Datadog
+
+## Overview
+
+The following describes how to configure Datadog to gather metrics from Artifactory and Xray.
+
+## Requirements
+
+* Kubernetes Cluster
+* Artifactory and/or Xray installed via [JFrog Helm Charts](https://github.com/jfrog/charts)
```

Comment:
links need to be updated to this format `[JFrog Helm Charts][1]`

with the corresponding number at the bottom of the page
`[1]: https://github.com/jfrog/charts`

I would also link out the the rest of the requirements points